src/ui/public/state_management/__tests__/state.js

```diff
@@ -67,29 +97,29 @@ describe('State Management', function () {
 
   describe('Search', function () {
     it('should save to $location.search()', function () {
-      let state = new State('_s', { test: 'foo' });
+      const { state, getUnhashedSearch } = setup({ initial: { test: 'foo' } });
       state.save();
-      let search = $location.search();
+      let search = getUnhashedSearch(state);
       expect(search).to.have.property('_s');
       expect(search._s).to.equal('(test:foo)');
     });
 
     it('should emit an event if changes are saved', function (done) {
-      let state = new State();
+      const { state, getUnhashedSearch } = setup();
       state.on('save_with_changes', function (keys) {
         expect(keys).to.eql(['test']);
         done();
       });
       state.test = 'foo';
       state.save();
-      let search = $location.search();
+      let search = getUnhashedSearch(state);
```

Comment:
similar to the other unused var comments. as you can tell, I like when there's less code. ;)